mico-core/src/main/java/io/github/ust/mico/core/web/ApplicationController.java

```diff
@@ -306,7 +298,7 @@
     }
     
     @PutMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}/services/{" + SERVICE_SHORT_NAME + "}")
-    public ResponseEntity<Resource<MicoApplication>> updateServiceDeploymentInformation(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName,
+    public ResponseEntity<Resource<MicoServiceDeploymentInfoDTO>> updateServiceDeploymentInformation(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName,
```

Comment:
replace 'services' by something like 'deploymentInformation' in the mapping

Reply:
Do the same in the GET endpoint

Reply:
Done, now it's
`GET /applications/{shortName}/{version}/deploymentInformation/{serviceShortName}` and
`PUT /applications/{shortName}/{version}/deploymentInformation/{serviceShortName}`